remix-tests/tests/examples_2/simple_storage_test.sol

```diff
@@ -13,14 +13,11 @@ contract MyTest {
     i += 1;
   }
 
-  function initialValueShouldBe100() public view returns (bool) {
-    //return Assert.equal(foo.get(), 100, "initial value is not correct");
-    return foo.get() == 100;
+  function initialValueShouldBe200() public returns (bool) {
+    return Assert.equal(foo.get(), 200, "initial value is not correct");
   }
 
-  function initialValueShouldBe200() public view returns (bool) {
-    //return Assert.equal(foo.get(), 200, "initial value is not correct");
-    return foo.get() == 200;
+  function valueShouldBe100() public returns (bool) {
```

Comment:
any specific reason to remove 'initial' from function name

Reply:
In this particular test initial value was set to `100` but later on 2nd iteration it should be `200`.

Reply:
yes, with that, initialValueShouldBe100 looks more reasonable as function name IMO.

Reply:
Yes it does. But for some unknown reason in my local machine the code executes in the following order and works fine
```
function initialValueShouldBe100()
function finalValueShouldBe200()
```
but in circleci the order switches their position and results in failure. Not sure why. I think we can ignore this test for now.

Reply:
hmm okay